aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -139,6 +139,19 @@ module aptos_token_objects::aptos_token {
         property_types: vector<String>,
         property_values: vector<vector<u8>>,
     ) acquires AptosCollection, AptosToken {
+        mint_inner(creator, collection, description, name, uri, property_keys, property_types, property_values);
+    }
+
+    public fun mint_inner(
```

Comment:
This completely changes the intention of AptosToken, which is meant to be not extensible. It's basically the entry level token that cannot be extended. If users want custom token, they should build their own instead of on top of AptosToken.

Reply:
Then I would like to know why we disallow that.